subsys/net/l2/ethernet/gptp/Kconfig

```diff
@@ -103,6 +103,13 @@ config NET_GPTP_CLOCK_ACCURACY
 	default 0x31 if NET_GPTP_CLOCK_ACCURACY_GT_10S
 	default 0xfe
 
+config NET_GPTP_STACK_SIZE
+	int "gPTP thread stack size"
+	default 2048
+	help
```

Comment:
Does it make sense to give separate defaults for such options that require a larger stack if they can be enumerated? Maybe you have some very specific options in mind? This way users of such options don't have to know about the internal requirement for a larger stack.

Reply:
I have `CONFIG_LOG_MODE_IMMEDIATE` in mind. But it requires an increase in all the stacks where there are things being printed. I don't see a default value for that in other stack sizes definition, so I don't want to have a particular case here for no reason.

Reply:
agreed.